Probesys/bileto
# imp: Improve the form to open tickets

## Related issue(s)



https://github.com/Probesys/bileto/issues/116
https://github.com/Probesys/bileto/issues/148
https://github.com/Probesys/bileto/issues/243

## Changes



- Change flex-grow by flex property
- Set pointer cursor on <summary> elements
- Put renamed tickets titles between quotes
- Add a "type" field to the "new ticket" form
- Add priority fields to the "new ticket" form
- Hide fields of this form based on the permissions given to the user

## How to test manually



- go to the "new ticket" form with full permissions → check that you can choose the type and the priority of the ticket
- go to the "new ticket" form with only the "Create tickets" permission → check that you can only choose a title and add a message

## Checklist
